docs/tech_concepts_and_architecture_overview.md

```diff
@@ -44,12 +47,12 @@ For details, check the automatically generated documentation of the RestAPI and
 [`docs/generated`](./generated/)
 
 ## Database schema
+
 The database has a strongly typed schema. There are two types of classes; Things and Actions.
 
-From the perspective of the Database schema, these are identical, so we'll only dicuss the schema
-of this general class.
+Although the schemas are identical from the database schema perspective, they are separated to make it easier for end users to interact with the service (i.e., actions are verbs, things are nouns)
```

Comment:
How is this exactly easier than having just "classes"?
[TBBH](https://www.urbandictionary.com/define.php?term=Brutally%20Honest) I don't understand this. How can you expect other techies to understand this?

This does _not_ make it easier, only more complicated because of this distinction.

I'm not sure if this is the case, but a better reason would be to state that distinguishing between things and actions helps to do the fuzzy queries later on. It is definitely _not_ easier than having two disjunct schema's.

Reply:
I stayed away from all these value judgements on purpuse; just describing the harsh technical reality of what this project is doing.